arcade/sprite/base.py

```diff
@@ -346,26 +349,19 @@ def color(self) -> Color:
 
     @color.setter
     def color(self, color: RGBA255):
```

Comment:
This'll need an import change at the top, but I think we should do this while we're at it.
```suggestion
    def color(self, color: RGBOrA255):
```